client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -378,6 +390,10 @@ static String endpoint(String[] indices, String[] types, String endpoint) {
         return endpoint(String.join(",", indices), String.join(",", types), endpoint);
     }
 
+    static String endpoint(String[] indices, String endpoint) {
```

Comment:
(see comment thread: https://github.com/elastic/elasticsearch/pull/27019#discussion_r145629135)

Reply:
thanks for your comments, how about using the existing endpoint method then and calling it like this `endpoint(indices, Strings.EMPTY_ARRAY, "")` ?